ts/component/table-builder.ts

```diff
@@ -605,13 +603,17 @@ export class TableBuilder<ParentType, ChildType> {
     }
     const data = await response.json();
     this.load(data.items);
-    this.showLoaded(data);
+    if (!this.definition.disablePageControls) {
+      this.showLoaded(data);
+    }
```

Comment:
Move this check to inside `showLoaded`. Similar to `showLoading`, there will likely be parts of this that should be called in the future even when not showing the page controls